build/tasks/publish.js

```diff
@@ -106,28 +104,30 @@ gulp.task("npm:ui:readme", function () {
         .pipe(gulp.dest(npmFolder + "/clarity-ui"));
 });
 
-gulp.task("npm:ui", ["npm:ui:deliverables", "npm:ui:sources", "npm:ui:package", "npm:ui:readme"], function () {});
+gulp.task("npm:ui", ["npm:ui:bundles", "npm:ui:sources", "npm:ui:package", "npm:ui:readme"], function () {});
 
 /**
  * Preparing the clarity-angular package
  */
 
 /**
  * The deliverables for clarity-angular are:
- *   - the compiled js files in commonJS format
- *   - the minified JS bundle
+ *   - the compiled js files in es2015 format
```

Comment:
I'm a bit scared of this. I think the standard is ES5 with ES2015 module format only, not the full features. Otherwise you either force Clarity's user to use yet another transpiler to ES5, or they forget about it and their app will only run on a handful of browsers.

For instance, when I look at the AoT documentation, they do this in their `tsconfig-aot.json`:
```javascript
{
  "compilerOptions": {
    "target": "es5",
    "module": "es2015",
    ...
  }
}
```

I'm guessing there is a good chance that's what you're doing, but the comments and commit message should probably be less ambiguous when you say "es2015 format". I know it's a mouthful, but "es5 with es2015 module format" will avoid confusion.